### PR TITLE
Homework: max of vector

### PR DESCRIPTION
**Pull request by michalsoloducha**

### CODE REVIEW

**Comment by ziobron** [comment]
✅ CI Passed - Task: max-of-vector
🏅 Granted 5 XP for @michalsoloducha

**Comment by ziobron** [comment]
⏱ 3 XP extra for @michalsoloducha for on-time delivery

File: homework/max-of-vector/maxOfVector.hpp [hunk 1]
```diff
@@ -3,6 +3,10 @@
 #include <limits>
 
 int maxOfVector(const std::vector<int>& vec) {
-    // TODO: Implement me :)
-    return {};
+    // auto result = std::max_element(vec.begin(), vec.end());
+    int result = vec.front();
+    for (auto element : vec) {
+        if (element > result) result = element;
```
Comment (nauka-programowania-MA) [inline homework/max-of-vector/maxOfVector.hpp:9]: one command one line :)

**Review by nauka-programowania-MA — APPROVED** [review]

File: homework/max-of-vector/maxOfVector.hpp (hunk 1 above)
Comment (michalsoloducha) [reply homework/max-of-vector/maxOfVector.hpp:9]: I thought there is some kind of "max" command but not sure which and I couldn't make it work with max_element.

Comment (nauka-programowania-MA) [reply homework/max-of-vector/maxOfVector.hpp:9]: yes you can use `std::max_element()`. You just need to pass itertator to begin and end of vector and the derefenrece it.

return *std::max_element(vec.cbegin(), vec.cend());

**Pull request closed without merging**